dev/io.openliberty.checkpoint_fat_xmlws/fat/src/io/openliberty/checkpoint/jaxws/fat/EJBWSBasicTest.java

```diff
@@ -44,6 +48,8 @@
 @SkipIfCheckpointNotSupported
 public class EJBWSBasicTest {
 
+    private static final String SERVER_NAME = "EJBWSBasicServer";
+
     @Server("EJBWSBasicServer")
```

Comment:
Should use new constant for `SERVER_NAME` here

Reply:
Done!